src/pages/Details.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaMicrophone } from 'react-icons/fa';
+import { IoMdSettings, IoIosArrowBack } from 'react-icons/io';
+import { NavLink } from 'react-router-dom';
+import { fetchDetails } from '../redux/Details';
+import '../styles/Details.css';
+
+const Details = () => {
+  const details = useSelector((state) => state.details);
+  const urlElement = window.location.href.split('/')[5];
+  const country = window.location.href.split('/')[4];
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchDetails(urlElement));
+  }, [urlElement, dispatch]);
+
+  return (
+    <>
+      <nav className="navBar">
+        <NavLink to={`/companies/${country}`}>
+          <IoIosArrowBack />
+        </NavLink>
+        <div className="settings">
+          <FaMicrophone />
+          <IoMdSettings />
+        </div>
+      </nav>
+      {details.map((detail) => (
+        <div className="info" key={detail.symbol}>
+          <div className="price-container">
+            <img alt="company-img" src={detail.image} />
+            <div className="price-box">
+              <h2>{`${detail.companyName}`}</h2>
+              <h4>{`Price: $${detail.price}`}</h4>
+            </div>
+          </div>
+          <div>
+            <h3>{`Currency: ${detail.currency}`}</h3>
+            <h3>{`Industry: ${detail.industry}`}</h3>
+            <h3>{`Sector: ${detail.sector}`}</h3>
+            <h3>{`CEO: ${detail.ceo}`}</h3>
+          </div>
+        </div>
+      ))}
```

Comment:
- [ ] According to the requirements, You should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including: 
* Typography: font face, size, and weight
* Layout: composition and space between elements.

So Kindly update the design for the details page to look similar to the original design 💯

| Your Design  | Original Design  |   
| --- | --- |
| <img src="https://user-images.githubusercontent.com/45231276/205421252-aef03101-3307-4f37-8e32-bf68e9b6f803.png" width="500" />  | <img src="https://user-images.githubusercontent.com/45231276/205421212-9f967f45-c7ce-437f-aa22-7dc41c3c1468.PNG" width="500" /> |

Reply:
I fixed the layout and fonts, thank you for your feedback!